distribution/src/main/release/samples/pom.xml

```diff
@@ -261,24 +261,24 @@
                     <version>1.3.1</version>
                 </dependency>
                 <dependency>
-                    <groupId>javax.xml.ws</groupId>
-                    <artifactId>jaxws-api</artifactId>
-                    <version>2.3.0</version>
+                    <groupId>jakarta.xml.ws</groupId>
+                    <artifactId>jakarta.xml.ws-api</artifactId>
+                    <version>2.3.2</version>
                 </dependency>
                 <dependency>
```

Comment:
I think we could remove `javax.activation-api`, `jakarta.activation-api` replaces it.

Reply:
Definitely, my plan was to do it along with Spring upgrade afterwards. But we can also do it now, it's hard to separate these things.